src-ts/utils/account/styles.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../lib/styles';
+
+.page-container {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    margin: 0 $pad-xxxxl;
+
+    h1 {
+        color: #2A2A2A;
+        border-bottom: 1px solid $black-10;
+    }
+
+    .page-content {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        margin: $pad-xxxxl -#{$pad-md} 0;
+
+        @include ltemd {
+            flex-direction: column;
+        }
+
+        .card {
+            width: calc(33.3% - #{$pad-xxl});
+            margin: 12px;
+
+            @include ltemd {
+                width: calc(100% - #{$pad-xxl});
+            }
+
+            p {
+                word-break: break-all;
```

Comment:
this style should still apply when it's nested directly under page-content and doesn't have the .card style:

```
.page-content {
    ...

    p {
        ...
    } 
}
```